spec/rubocop/cop/variable_force/assignment_spec.rb

```diff
@@ -125,8 +125,8 @@ def some_method
         RUBY
       end
 
-      it 'returns splat node' do
-        expect(assignment.meta_assignment_node.type).to eq(:splat)
+      it 'returns masgn node' do
+        expect(assignment.meta_assignment_node.type).to eq(:masgn)
```

Comment:
`splat` is not a type of assignment, so it makes sense to me that this returns `masgn` instead now. I think the test was just wrong previously.